src/components/CommentTeaser/CommentTeaser.js

```diff
@@ -127,12 +120,23 @@ export const CommentTeaser = ({
     Link
   }
 
+  const linkEMRule = useMemo(
+    () =>
+      css({
+        '& em': {
+          background: colorScheme.getCSSColor('hover'),
```

Comment:
hover is too weak here (was `primaryBg`)
<img width="1167" alt="Screenshot 2020-11-03 at 00 27 59" src="https://user-images.githubusercontent.com/410211/97930353-78c7d380-1d6b-11eb-9206-a4926b06baa8.png">
(before and after)

maybe `alert`?